src/size.rs

```diff
@@ -69,6 +69,11 @@ impl Size {
 
     /// Returns a new size bounded by `min` and `max.`
     ///
+    /// # Panics
```

Comment:
I think it's fine if we follow std semantics for panic, and as you say we can revisit if this ends up being a problem. cc @raphlinus in case he has any opinions.